line-colors.csv

```diff
@@ -53,24 +53,24 @@ db-regio-nordost,RE66,db-regio-ag-nordost,re-66,#007734,#ffffff,,rectangle
 db-regio-nordost,RE7,db-regio-ag-nordost,re-7,#007734,#ffffff,,rectangle
 db-regio-sudost,RE14,db-regio-ag-sudost,re-14,#a98956,#ffffff,,rectangle
 db-regio-sudost,S1,db-regio-ag-sudost,4-800486-1,#f5de2b,#000000,,pill
+db-regio-sudost,S10,db-regio-ag-sudost,4-800486-10,#ffffff,#000000,#f5de2b,pill
```

Comment:
10 after 1? Yes, I know it's technically correct, but it feels wrong.

For some regions and lines it's fine because line S47 is a shorter or slightly modified version of line S4, but line S10 is probably just the next line after S9 and not a variant of line S1. I'm really not sure if this is the best way to order line numbers...

Reply:
I agree, the numbers should be sorted numerically.

Reply:
The sorting in its current state is absolutely inconsistent and varies between different parts of the dataset. In my opinion this should be fixed. As the technically correct way of sorting doen't seem to be preferred here, I'd like to know how to proceed.